src/main/java/com/example/clinic/controllers/ServiceController.java

```diff
@@ -0,0 +1,68 @@
+package com.example.clinic.controllers;
+
+import com.example.clinic.dtos.ServiceDto;
+import com.example.clinic.sevices.ServiceService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping(value = "/v1/services")
+@RequiredArgsConstructor
+public class ServiceController {
+
+    private final ServiceService serviceService;
+
+    @PostMapping
+    @ResponseStatus(HttpStatus.CREATED)
+    public ServiceDto create(
+            @RequestBody ServiceDto service,
+            @RequestParam Long idClinic) {
```

Comment:
Id of clinic should be in url but not in request param according to the best practices of REST